### ISSUE TITLE
providing gitlab url with / fails 

### ISSUE THREAD

**Issue by niklas-dahl** [jdalrymple/node-gitlab-api#48]
using 

```
const GitlabAPI = gitlab({
  url: 'https://PRIVATE_GITLAB_DOMAIN.com',
  token: apiToken,
});
```
works, but when using `url: 'https://PRIVATE_GITLAB_DOMAIN.com/',` (with `/` at the end) fails without a descriptive error, just a `404 Not Found`.

I would expect that both ways would work..
Thanks.

**Comment by jdalrymple** [comment]
Yup it should thats weird. I'll look into it :)

**Comment by jdalrymple** [comment]
Found the problem

**Comment by jdalrymple** [comment]
Will release later today!

**Comment by jdalrymple** [comment]
Just released under 2.2.5. Should work for you now :)

**Comment by jdalrymple** [comment]
Fixed in 2.2.6 (switched to the url-join library)
